data/core/encyclopedia/geography.cfg

```diff
@@ -152,34 +170,49 @@ Over the River Aethen, south of Fort Tahn, is a Wesnothian frontier region. It i
 [topic]
     id=heart_mountains
     title= _ "Heart Mountains"
-    text= _ "A virtually impassable barrier between the <ref>dst='arkan_thoria' text='river'</ref> country and the <ref>dst='far_north' text='Northern Plains'</ref>."
+    text= _ "A virtually impassable barrier between the <ref>dst='arkan_thoria' text='river'</ref> country and the <ref>dst='far_north' text='Northern Plains'</ref>. The local orcs call them the Haggid-Dargor. The powerful Whitefang Clan in the region is named after the eastern edge of the mountains, which they call the Whitefang Mountains for their distinctive shape."
```

Comment:
A po hint saying  which campaign these names from would be useful not just for translators checking that they use the same translation as the campaign.

```
#po: The Whitefang Clan’s primary description is in Descent into Darkness.
#po: Wesnoth 1.9.4 added Haggid-Dargor to the encyclopedia, but it has no other mention in mainline.
```